<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analizador de Encuestas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>../assets/Logo App.png</activeon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
     QWidget {
         background-color: #1a1b2e;
         color: white;
         font-family: 'Consolas', 'Courier New', monospace;
         font-size: 12pt;
     }
     QFrame#sidePanel {
         background-color: #27293d;
         border-radius: 15px;
         margin: 10px;
         padding: 10px;
     }
     QPushButton {
         background-color: #2196F3;
         color: white;
         border-radius: 8px;
         padding: 15px;
         text-align: center;
         margin: 5px;
         font-weight: bold;
         font-size: 14pt;
         font-family: 'Consolas', 'Courier New', monospace;
         border: 2px solid transparent;
     }
     QPushButton:hover {
         background-color: #1976D2;
         border: 2px solid #2196F3;
     }
     QPushButton:disabled {
         background-color: #4f4f4f;
         color: #8e8e8e;
         border: 2px solid transparent;
     }
     QLabel {
         color: #00b4d8;
         font-weight: bold;
         font-size: 13pt;
     }
     QFrame#graphContainer {
         background-color: #27293d;
         border-radius: 15px;
         margin: 10px;
         padding: 15px;
         min-height: 400px;
     }
     QTextEdit {
         background-color: #27293d;
         border: 1px solid #3a3b4d;
         border-radius: 8px;
         padding: 15px;
         color: #00b4d8;
         font-size: 14pt;
         font-family: 'Consolas', 'Courier New', monospace;
     }
     QLabel#lblTitle {
         font-size: 28pt;
         color: #00b4d8;
         margin-bottom: 20px;
         font-family: 'Consolas', 'Courier New', monospace;
     }
     QLabel#lblTipoGrafica {
         font-size: 16pt;
         color: #00b4d8;
         margin-top: 20px;
     }
     QLabel#lblFooter {
         color: #00b4d8;
         font-size: 12pt;
         margin-top: 10px;
     }
     QLabel#lblLogo {
         margin: 20px;
     }
     QLabel#lblNoData {
         color: #00b4d8;
         font-size: 18pt;
     }
    </string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="sidePanel">
      <layout class="QVBoxLayout" name="verticalLayoutSide">
       <item>
        <widget class="QLabel" name="lblTitle">
         <property name="text">
          <string>Analizador de
Encuestas</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCargarCSV">
         <property name="text">
          <string> Cargar CSV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResumen">
         <property name="text">
          <string> Mostrar Resumen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameGraficas">
         <layout class="QVBoxLayout" name="verticalLayoutGraficas">
          <item>
           <widget class="QLabel" name="lblTipoGrafica">
            <property name="text">
             <string>Tipo de Gráfica:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGraficaPastel">
            <property name="text">
             <string> Gráfica de Pastel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGraficaBarras">
            <property name="text">
             <string> Gráfica de Barras</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGuardarGrafica">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string> Guardar Gráfica</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblLogoFooter">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string>margin: 10px;</string>
         </property>
         <property name="pixmap">
          <pixmap>../assets/Logo Desarrollador.jpeg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFooter">
         <property name="text">
          <string>Desarrollado por Blanco Dev</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="graphContainer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>450</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutGraph">
       <item>
        <widget class="QTextEdit" name="txtResumen">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblNoData">
         <property name="styleSheet">
          <string>font-size: 16pt; color: #6c757d;</string>
         </property>
         <property name="text">
          <string>Carga un archivo CSV y selecciona un tipo de gráfica para comenzar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
